ssc/cmod_battery.cpp

```diff
@@ -1352,6 +1357,39 @@ void battstor::initialize_automated_dispatch(std::vector<ssc_number_t> pv, std::
                     }
                 }
             }
+
+            // All of these will be false for FOM, so load will not be populated
+            if (load_look_ahead) {
+                if (load.size() != 0)
+                {
+                    for (size_t idx = 0; idx != nrec; idx++) {
+                        load_prediction.push_back(load[idx]);
+                    }
+                }
+            }
+            else if (load_look_behind) {
+                // day one is zeros
+                for (size_t idx = 0; idx != 24 * step_per_hour; idx++)
+                {
+                    load_prediction.push_back(0);
```

Comment:
why isn't this from the last 24 hours of the load forecast?

Reply:
I was matching the convention from the weather file forecast: https://github.com/NREL/ssc/blob/fcff96bd16bf1a8dbc38966a94bca69d2ab99ec0/ssc/cmod_battery.cpp#L1328 is it worth changing both?

Reply:
I think so. I'd imagine if we're in year 2, and if we were doing lookahead, we'd use jan data for dec 31, and if doing lookbehind, we'd do the counterpart. This keeps continuity. What do you think?